src/app/python/core.py

```diff
@@ -477,21 +476,50 @@ def _task_program_launch(self, mainpath, args, caller):
     """
     if caller not in self._project_execs:
       if mainpath in self._project_files:
-        # The -u switch forces subprocess to be unbuffered
-        # It is better than subprocess.bufsize parameter
-        # since it does not seems to always work
-        cmd = ['python', '-u', self._project_src_path+mainpath] + args.split()
-        exec_process = subprocess.Popen(args=cmd,
-                                        stdin=subprocess.PIPE,
-                                        stdout=subprocess.PIPE,
-                                        stderr=subprocess.STDOUT,
-                                        env=dict(),  # For security purposes
-                                        )
-
-        # Save execution
-        self._project_execs[caller] = Core.Exec(exec_process, mainpath, args)
-        # Notify started
-        self._notify_event(lambda l: l.notify_program_started(mainpath, args, caller))
+        # Where copied files will be written
+        exec_path = os.path.join(self._project_exec_path, caller)
+        print "exec_path", exec_path
+
+        # Write to disk latest changes
+        project_nodes = (node for (node, is_dir) in self._impl_get_project_nodes())
+
+        try:
+          for filenode in project_nodes:
+            # Create any parent needed directories
+            exec_filenode_path = exec_path+filenode
+            exec_filenode_dir = os.path.dirname(exec_filenode_path)
+            if not os.path.exists(exec_filenode_dir):
+              os.makedirs(exec_filenode_dir)
+
+            # Create file
+            with open(exec_filenode_path, "w") as copied_file:
+              # Not reading from disk to get the lastest version
+              _, content, _ = self._impl_get_file_content(filenode)
+              copied_file.write(content)
+              copied_file.flush()  # Make sure text gets written
+
+          # The -u switch forces subprocess to be unbuffered
+          # It is better than subprocess.bufsize parameter
+          # since it does not seems to always work
+          cmd = ['python', '-u', exec_path+mainpath] + args.split()
```

Comment:
The flag would be -B to avoid generating .pyc and .pyo
It won't be necessary since execution will be into another directory than the one displayed for edit